tidb-cloud/tidb-cloud-glossary.md

```diff
@@ -53,7 +53,7 @@ Project members are users who are invited to join one or more projects of the or
 
 ### Recycle Bin
 
-The place where the data of deleted clusters with valid backups is stored. The deleted data or backup files will be retained here for a certain duration of time based on your plan. To avoid data loss, remember to restore the data to a new cluster in time. Note that if a cluster has no backup file, the deleted cluster will not be displayed here.
+The place where the data of deleted clusters with valid backups is stored. Once a backed up cluster is deleted, the existing backup files of the cluster are moved to the recycle bin. For backup files from automatic backups, the recycle bin can retain them for 7 days. For backup files from a manual backup, there is no expiration date. To avoid data loss, remember to restore the data to a new cluster in time. Note that if a cluster **has no backup file**, the deleted cluster will not be displayed here.
```

Comment:
```suggestion
The place where the data of deleted clusters with valid backups is stored. Once a backed up cluster is deleted, the existing backup files of the cluster are moved to the recycle bin. For backup files from automatic backups, the recycle bin can retain them for 7 days. For backup files from manual backups, there is no expiration date. To avoid data loss, remember to restore the data to a new cluster in time. Note that if a cluster **has no backup file**, the deleted cluster will not be displayed here.
```